unity-client/Assets/Scripts/MainScripts/DCL/Components/Avatar/AvatarRenderer.cs

```diff
@@ -315,11 +313,14 @@ private IEnumerator LoadAvatar()
             yield return new WaitUntil(() => bodyShapeController.isReady && wearableControllers.Values.All(x => x.isReady));
 
 
-            eyesController.Load(bodyShapeController, model.eyeColor);
-            eyebrowsController.Load(bodyShapeController, model.hairColor);
-            mouthController.Load(bodyShapeController, model.skinColor);
+            eyesController?.Load(bodyShapeController, model.eyeColor);
+            eyebrowsController?.Load(bodyShapeController, model.hairColor);
+            mouthController?.Load(bodyShapeController, model.skinColor);
 
-            yield return new WaitUntil(() => eyebrowsController.isReady && eyesController.isReady && mouthController.isReady);
+            yield return new WaitUntil(() =>
+                (eyebrowsController == null || (eyebrowsController != null && eyebrowsController.isReady)) &&
+                (eyesController == null || (eyesController != null && eyesController.isReady)) &&
+                (mouthController == null || (mouthController != null && mouthController.isReady)));
```

Comment:
Is it possible to simplify this checks like `yebrowsController?.isReady && eyesController?.isReady && ...` not entirely sure if possible but it sounds like it might :P

Reply:
I tried to simplify but the OR/AND operators don't let us to use `bool?` operators. Finally, this is the more clear way I found, sorry!